dist/spec.yaml

```diff
@@ -11418,6 +11750,154 @@ definitions:
     type: string
     x-okta-tags:
       - Application
+  OrgContactType:
+    enum:
+      - BILLING
+      - TECHNICAL
+    type: string
+    x-okta-tags:
+      - Org
+  OrgContactTypeObj:
+    properties:
+      _links:
+        additionalProperties:
+          type: object
+      contactType:
+        $ref: '#/definitions/OrgContactType'
+    type: object
+    x-okta-crud:
+      - alias: read
+        arguments: []
+        operationId: getOrgContactTypes
+    x-okta-tags:
+      - Org
+  OrgContactUser:
+    properties:
+      _links:
+        additionalProperties:
+          type: object
+        readOnly: true
+        type: object
+      userId:
+        type: string
+    type: object
+    x-okta-operations:
+      - alias: getContactUser
+        arguments: []
+        operationId: getOrgContactUser
```

Comment:
These lines are redundant. In Java SDK this will be interpreted as operation `getOrgContactUser` belongs to the `OrgContactUser` object which is incorrect.